src/clarity-angular/emphasis/alert/alerts.html

```diff
@@ -0,0 +1,6 @@
+<clr-alerts-pager
+        *ngIf="multiAlertService.count > 1"
+        [ngClass]="currentAlertType"
```

Comment:
Why do you need to add the class to the pager itself when you already have it on the `alerts` host? It would be simpler to just use CSS once it's on the parent than rely on `ngClass`.

EDIT: Actually after re-checking the CSS, I don't see you using this class anymore, so you can just remove it I guess?

Reply:
Yeah, removed since no longer needed.